### ISSUE TITLE
Replaced Token doesn't work inside DataDivision

### ISSUE THREAD

**Issue by smedilol** [TypeCobolTeam/TypeCobol#599]
Consider this example:
```cobol
       IDENTIFICATION DIVISION.
       PROGRAM-ID. TCOZCHKP.
       data division.
       working-storage section.
       REPLACE                                                          
           ==:LONGUEUR-IDOFUS:==      BY ==0900==.                                                     
       01  var1         PIC X(:LONGUEUR-IDOFUS:).                                                                           
      
       PROCEDURE DIVISION.
           move :LONGUEUR-IDOFUS: to var1
           .
      
       END PROGRAM TCOZCHKP.
```

ConsumedToken for CodeElement of Var1 contains `:LONGUEUR-IDOFUS:`
ConsumedToken for CodeElement of move contains `0900`

It seems Token are not replaced correctly when inside DataDivision.

**Comment by smedilol** [comment]
@mayanje I assigned you because it seems to be the same bug as #1753

**Comment by rooksdo** [comment]
Fixed by #1798
